.circleci/config.yml

```diff
@@ -570,7 +558,6 @@ workflows:
             - noir-x86_64
             - noir-arm64
           <<: *defaults
-      - noir-tests: *defaults
```

Comment:
IIUC noir's tests weren't blocking PRs so I didn't add it to the `merge-check` in GA either.

Reply:
This is likely an oversight

Reply:
Ok, I'll add it to GA's merge-check